motor_control/motor_control.cpp

```diff
@@ -0,0 +1,65 @@
+#include "motor_control.h"
+
+Adafruit_MotorShield AFMS = Adafruit_MotorShield();
+Adafruit_DCMotor *myMotorL = AFMS.getMotor(1);
+Adafruit_DCMotor *myMotorR = AFMS.getMotor(4);
+
+void Mmotor::begin() {
+	AFMS.begin();
+}
+
+void Mmotor::setMotors(int mL, int mR) {
+	Adafruit_DCMotor *myMotorL = AFMS.getMotor(mL);
+	Adafruit_DCMotor *myMotorR = AFMS.getMotor(mR);
+}
+
+void Mmotor::setMotorSpeed(int mL, int mR) {
+	//Sets speed of motors
+	myMotorL->setSpeed(mL);
+	myMotorR->setSpeed(mR);
+}
+
+void Mmotor::runMotor(bool boo, char motor) {
```

Comment:
Can you indicate somewhere with a comment that b is both, l is left, r is right?